### ISSUE TITLE
Pls expose useFormContext

### ISSUE THREAD

**Issue by Achaconm2001** [ivan-dalmet/formiz#74]
Im trying to made my own version of the FormStep because I need the inputs to stay in the dom instead of just disappearing (https://github.com/ivan-dalmet/formiz/blob/master/packages/core/src/FormizStep.tsx#L94). 

This is my code, its almost the same except for the render part, the error happends on the last import (import { useFormContext } from "@formiz/core/dist/Formiz") because the useFormContext isn't expose at the root:
```typescript
import { createContext, FC, useContext, useEffect, useRef, useState, RefObject } from "react"
import { Collapse } from "@chakra-ui/react"
import { FormizStepProps, StepState } from "@formiz/core/dist/types/step.types";
import { FormState } from "@formiz/core/dist/types/form.types";
import { useFormContext } from "@formiz/core/dist/Formiz";

// https://github.com/ivan-dalmet/formiz/blob/master/packages/core/src/FormizStep.tsx

export const getFormUniqueId = (): string => `form-${Math.random().toString(36).substr(2, 9)}`;

export const defaultFormState: FormState = {
  id: getFormUniqueId(),
  resetKey: 0,
  isSubmitted: false,
  isValidating: false,
  isValid: true,
  isPristine: true,
  steps: [],
  initialStepName: null,
  navigatedStepName: null,
};


export const StepContext = createContext<any>({});
export const useStepContext = (): any => useContext(StepContext);

export const useRefValue = (value: any): RefObject<any> => {
  const ref = useRef(value);
  ref.current = value;
  return ref;
};

export const FormStep: FC<FormizStepProps> = ({ as: Tag = "div", ...props }) => {

  const {
    formStateRef,
    actions,
    subjects,
  } = useFormContext();

  const [formState, setFormState] = useState(formStateRef?.current ?? defaultFormState);
  const [state, setState] = useState<Partial<StepState>>({
    name: props.name,
    label: props.label,
    isVisited: false,
    order: props.order ?? 0,
  });


  const isActive = formState.navigatedStepName
    ? formState.navigatedStepName === props.name
    : formState.initialStepName === props.name;

  const actionsRef = useRefValue(actions);
  const subjectsRef = useRefValue(subjects);

  useEffect(() => {
    const subscription = subjectsRef.current.onFormUpdate
      .subscription
      .subscribe(setFormState);
    return () => subscription.unsubscribe();
  }, [subjectsRef]);

  useEffect(() => {
    if (isActive && !state.isVisited) {
      setState((prevState) => ({ ...prevState, isVisited: true }));
    }
  }, [isActive, state.isVisited]);

  useEffect(() => {
    actionsRef.current.updateStep({
      ...state,
      isEnabled: props.isEnabled,
    });
  }, [actionsRef, state, props.isEnabled]);

  useEffect(() => () => {
    actionsRef.current.unregisterStep(props.name);
  }, [actionsRef, props.name]);


  return (
    <StepContext.Provider value={{ name: props.name, }}>
      <Tag
        style={{
          ...props.style,
        }}
        {...props}
      >
        <Collapse in={isActive}>
          {props.children}
        </Collapse>
      </Tag>
    </StepContext.Provider>
  );
}

export default FormStep
```

**Comment by ivan-dalmet** [comment]
Hello @Achaconm2001 
We just published a new version `1.1.0` of `@formiz/core`
This version do not export the `useFormContext` as asked. Because exposing internal stuff that we can break later does not feel right.

BUT, we added an `autoHide` property on `<FormizStep>` that you can pass to `false` to handle your use case! :) 
So you can do something like that:
```jsx
<Collapse in={form?.currentStep?.name === 'step1'}>
  <FormizStep autoHide={false} name="step1">
    {/* Your fields here */}
  </FormizStep>
</Collapse>
```

You can of course create your own wrapper component
```jsx
const MyFormizStep = (props) => {
  const form = useForm({ subscribe: 'form' });
  return (
    <Collapse in={form?.currentStep?.name === props.name}>
      <FormizStep autoHide={false} {...props} />
    </Collapse>
  );
}

const MyForm = () => {
  return (
    <Formiz>
      <MyFormizStep name="step1">
        {/* Your fields here */}
      </MyFormizStep>
      <MyFormizStep name="step2">
        {/* Your fields here */}
      </MyFormizStep>
    </Formiz>
  )
}
```

[See Documentation](https://formiz-react.com/docs/core/formiz-step/#autohide)

I hope this can help 😊
If we didn't understand the need, feel free to say so 😅

**Comment by Achaconm2001** [comment]
Thanks, that was exactly what I needed :D
